packages/docmanager-extension/src/index.tsx

```diff
@@ -262,7 +262,7 @@ const docManagerPlugin: JupyterFrontEndPlugin<void> = {
       const autosaveInterval = settings.get('autosaveInterval').composite as
         | number
         | null;
-      docManager.autosaveInterval = autosaveInterval || 120;
```

Comment:
For ease of changing this again in the future, it would be better to make this a symbolic constant (e.g., `DEFAULT_AUTOSAVE_INTERVAL`).

Reply:
done @JasonWeill